Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -280,44 +274,10 @@ AppDelegateViewModelOutputs {
           .map { _ in token }
       }
 
-    let remoteNotificationFromLaunch = self.applicationLaunchOptionsProperty.signal.skipNil()
-      .map { _, options in
-        options?[UIApplication.LaunchOptionsKey.remoteNotification] as? [AnyHashable: Any] }
-      .skipNil()
-
-    let localNotificationFromLaunch = self.applicationLaunchOptionsProperty.signal.skipNil()
-      .map { _, options in
-        options?[UIApplication.LaunchOptionsKey.localNotification] as? UILocalNotification }
-      .map { $0?.userInfo }
+    let deepLinkFromNotification = self.remoteNotificationProperty.signal.skipNil()
```

Comment:
This is a lot simpler now that we don't care about whether the app state is active.